Tests/InstallationTests/CommonFiles/RevenueCat-Swift.h

```diff
@@ -251,6 +251,100 @@ SWIFT_CLASS("_TtC10RevenueCat25CacheableNetworkOperation")
 @interface CacheableNetworkOperation : NetworkOperation
 @end
 
+@class NSString;
+@class RCConfigurationBuilder;
+
```

Comment:
I see you're not using Xcode 13.4.0 😛  (it's not a big deal).

Reply:
Uhhh... downloading now 😉 

Should I regenerate this or something? 🤷‍♂️

Reply:
Nope it doesn't matter! I just noticed that when I generated it it changed the Swift version hehe.
But it's not important.

Reply:
I did! :)